src/libraries/System.Net.NetworkInformation/src/System/Net/NetworkInformation/SystemIPGlobalProperties.cs

```diff
@@ -380,7 +380,7 @@ public override IcmpV6Statistics GetIcmpV6Statistics()
         public override async Task<UnicastIPAddressInformationCollection> GetUnicastAddressesAsync()
         {
             // Wait for the address table to stabilize.
-            var tcs = new TaskCompletionSource<bool>(TaskContinuationOptions.RunContinuationsAsynchronously);
+            var tcs = new TaskCompletionSource<bool>(TaskCreationOptions.RunContinuationsAsynchronously);
```

Comment:
Another analyzer warning catched on build: `error CA2247: Argument contains TaskContinuationsOptions enum instead of TaskCreationOptions enum. `

Reply:
Nice!